WooCommerce/UITestsFoundation/Screens/MyStore/MyStoreScreen.swift

```diff
@@ -3,11 +3,6 @@ import XCTest
 
 public final class MyStoreScreen: ScreenObject {
 
-    // TODO: Remove force `try` once `ScreenObject` migration is completed
-    public let tabBar = try! TabNavComponent()
-    // TODO: Remove force `try` once `ScreenObject` migration is completed
-    public let periodStatsTable = try! PeriodStatsTable()
-
```

Comment:
The removal of these lines actually blocks building the app for me:

<img width="1475" alt="Screenshot 2022-12-06 at 14 15 21" src="https://user-images.githubusercontent.com/73365754/205937055-0bf71c20-a9ff-4d96-ab5b-00e6cf7ac127.png">

I'm not sure how it worked for Buildkite. I think the comment meant removing the force try only, not the whole thing, because it's still in use.

Reply:
thanks for the review @pachlava! 

ah ok, i understood that the comment meant to only remove the force try (the `!`) but i thought that since it wasn't used by any methods on the screen, it would be safe to completely remove it. i did not realize that it was used to chain methods on the screenshot tests. 

i'm going to update the screenshots test to not be dependent on that instead. also, noticed a few things from the screenshots test:
1. i've accidentally created a duplicate in `.goToThisMonthTab()` which is the same as `switchToMonthsTab()` I'll do a cleanup PR after this one 
2. the screenshots are not working, again 😬 (the same behavior in trunk)

Reply:
screenshots test updated in https://github.com/woocommerce/woocommerce-ios/pull/8319/commits/d221ca744d09d92f9563398089a9b889b2b56b41

Reply:
Thanks for fixing these parts @jostnes!

> the screenshots are not working, again 😬 (the same behavior in trunk)

Yep, my initial concern was that the errors in them blocked the app from being built. And it's all good now in this respect. In the context of the screenshot test not "passing": I think it's not in scope of this PR, and the PR is not affecting it.